rosa_architecture/rosa-understanding.adoc

```diff
@@ -5,7 +5,55 @@ include::_attributes/attributes-openshift-dedicated.adoc[]
 :context: rosa-understanding
 toc::[]
 
-Learn about {product-title} (ROSA) access, supported consoles, consumption experience, and integration with Amazon Web Services (AWS) services.
+Learn about {product-title} (ROSA), supported consoles, consumption experience, and integration with Amazon Web Services (AWS) services.
 
-include::modules/rosa-understanding.adoc[leveloffset=+1]
-include::modules/rosa-using-sts.adoc[leveloffset=+2]
+== About ROSA
+
+ROSA is a fully-managed turnkey application platform that allows you to focus on what matters most, delivering value to your customers by building and deploying applications. Red Hat and AWS Site reliability engineering (SRE) experts manage the underlying platform so you do not have to worry about the complexity of infrastructure management. ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to further accelerate the building and delivering of differentiating experiences to your customers.
+
+This service is subscribed directly from your AWS account.
+
+After the clusters are created, you can operate your clusters with the OpenShift web console or through OpenShift Cluster Manager. The ROSA service also uses OpenShift APIs. These tools provide a standardized OpenShift experience to use your existing skills and tools knowledge.
```

Comment:
seems that we should also add "...also uses OpenShift APIs **and the OpenShift CLI**".